### PR TITLE
Add event debrief agenda

### LINKED ISSUE #455: Create template agenda for event debriefs
Staff members recently attended DecidimFest in Barcelona. This was a great meeting, with many potential opportunities and next steps to follow up on. On trying to hold a debrief meeting, we identified we should develop a solid and reusable agenda for these kind of meetings. 

We agreed to make a template agenda, and then rerun our DecidimFest debrief meeting with this agenda.

### PR DESCRIPTION
**Pull request by clausmullie**
fixes #455

-----
[View rendered activities/index.md](https://github.com/publiccodenet/about/blob/Add-staff-meetings-index/activities/index.md)
[View rendered activities/staff-meetings/event-debrief.md](https://github.com/publiccodenet/about/blob/Add-staff-meetings-index/activities/staff-meetings/event-debrief.md)
[View rendered activities/staff-meetings/index.md](https://github.com/publiccodenet/about/blob/Add-staff-meetings-index/activities/staff-meetings/index.md)

### CODE REVIEW

**Comment by clausmullie** [comment]
@Ainali could you give this a quick check and let me know what you think?

File: activities/staff-meetings/event-debrief.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+* developing our community
+* communications
+* improving About
+
+3) Next steps
+* generate overview of identified next steps 
+* assign next steps to a team of owners (at least 2 staff members)
+* define a timeframe, if relevant
+
+4) Decide if it would be useful to run a retro for this event
```
Comment (bvhme) [inline activities/staff-meetings/event-debrief.md]: This is a bit of a difficult thing in markdown but will parse differently in different engines and for that reason is not recommended. It is ambiguous as to whether this should be interpreted as an ordered list with sub bullets or multiple lists with headings.

I would recommend one of the following formatting options:

```markdown
1) A
  * b
  * c
2) D
  * e
```

Or alternatively:

```
## 1) A

* b
* c

## 2) D

* e
```

Additionally, headers or lists should always have whitespace around them.

I understand this is a bit of a hassle, however this is exactly what makes markdown portable, it forces you to use a limited set of structural elements that can then be designed in a system.

I can explain more if you want to.

File: activities/staff-meetings/index.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+type: Index
+explains: This index contains the resources and guides for staff to run and participate in staff meetings. 
+---
+
+# Staff meetings at the Foundation for Public Code
+
+[Event debrief](event-debrief.md)
+
+
```
[the remark this answers is not in the document]
Comment (clausmullie) [reply activities/staff-meetings/index.md]: Fixed with other PRs

File: activities/staff-meetings/event-debrief.md (hunk 1 above)
Comment (clausmullie) [reply activities/staff-meetings/event-debrief.md]: Thanks for the explanation! fixed in a different pull request

**Pull request merged**